docs/docs/faq.mdx

```diff
@@ -134,6 +134,20 @@ Ideally in this case you can redownload the notebook and use the new `gbstats` l
 If the queries in your notebook return sums and sums of squares as part of the `__stats` CTE but your notebook is still erroring, then you probably have an old `gbstats` version installed and need to update to 0.4.0 or later.
 You can download this from PyPI [here](https://pypi.org/project/gbstats) using `pip install gbstats` and ensure that your kernel uses that version of `gbstats`.
 
+## My features aren't refreshing as expected. What can I do?
+
+Our SDK's implement a stale-while-revalidate approach to cacheing with a configurable time-to-live (TTL) value.
+
+This means that if the feature payload is considered stale (i.e. more than the TTL amount of time has passed since it's been updated), **the next request will return the stale features** and refetch an update asynchronously so that on the next request, the features will be up to date. You can learn more about how our SDK's implement this in detail [here](/lib/build-your-own#fetching-and-caching-features).
+
+If you would like something more real-time than this stale-while-revalidate approach, you may want to consider implementing the [GrowthBook Proxy](https://docs.growthbook.io/self-host/proxy) on your self-hosted instance.
+
+## How do I configure environments in the SDK?
+
+When you create an SDK connection, it is linked to a specific environment. You can learn more about environments [here](/app/features#environments).
+
+---
```

Comment:
This doesn't add a line, just a bit extra whitespace, but this will help us when adding new items to this page to make sure we leave the "Can't find your question" item as the last item.

<img width="1169" alt="image" src="https://user-images.githubusercontent.com/113377031/235526884-480150c5-5d3a-4da9-92ce-36ab658f95d2.png">